frontend/fingertips-frontend/components/pages/chart/index.tsx

```diff
@@ -52,31 +51,33 @@ export function Chart({
           <br />
         </>
       ) : null}
-      {scatterData ? (
+      {indicatorsSelected.length > 1 ? (
```

Comment:
I think (based on [this page](https://confluence.collab.test-and-trace.nhs.uk/pages/viewpage.action?pageId=419245267)) this should only show the scatter chart if `indicatorsSelected.length` is exactly 2:
```suggestion
      {indicatorsSelected.length == 2 ? (
```